javascript/code-challenges/challenges-07.test.js

```diff
@@ -0,0 +1,359 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function called sortStarWarsCharacters that sorts the characters in the starWarsPeople array by height from tallest to shortest.
+------------------------------------------------------------------------------------------------ */
+
+let starWarsPeople = [
+  {
+    "name": "C-3PO",
+    "height": "167",
+    "eye_color": "yellow"
+  },
+  {
+    "name": "Luke Skywalker",
+    "height": "172",
+    "eye_color": "blue"
+  },
+  {
+    "name": "R2-D2",
+    "height": "96",
+    "eye_color": "red"
+  }
+];
+
+const sortStarWarsCharacters = (starWarsArr) => {
+  let newArr = starWarsArr.sort((firstName, secondName) => {
+    return  secondName.height - firstName.height;
+  });
+  return newArr;// Solution code here...
+}
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named removeThree that takes an index and an array. The function should removes three items in the array starting with the value at the index. 
+------------------------------------------------------------------------------------------------ */
+
+const removeThree = (idx, arr) => {
+ let newArr =  // Solution code here...
+};
+
```

Comment:
Hello there, you are on your way with these :)

For this one, your code challenges wants you to use array and string methods. So, splice(), slice() could work well here. arr.splice(starting index, number of elements to remove) or arr.slice(starting index, up until another index). See what you can get going 👍  

See here too: [slice(), splice(), split()](https://www.freecodecamp.org/news/lets-clear-up-the-confusion-around-the-slice-splice-split-methods-in-javascript-8ba3266c29ae/)